templates/cluster-template-rosa-machinepool.yaml

```diff
@@ -57,7 +57,7 @@ metadata:
   name: "${CLUSTER_NAME}-pool-0"
 spec:
   clusterName: "${CLUSTER_NAME}"
-  replicas: 1
+  replicas: 3
```

Comment:
We should add logic in the func [shouldUpdateRosaReplicas](https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/5386/files#diff-ca8ae33b4375b0269fe24617620c4d304d1882cb8d562c7b544331d272c7aec1R374) if the user set the replica to 1 in the machinePool saying ROSAMachinePool must have at least 3 replica per zone and machinePool replica changes will be ignored

Reply:
Setting 1 replica is ok, if there isnt the only machine pool. If there is the only one machine pool, number of replicas will not go lower than 2. If you try to set it to 1, reconcile with end with an error. I think that is OK behavior.

Reply:
hmm okay then, since we don't add default MP most likely we will not have this issue.  We may re-visit this when we work on adding default MP.